<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_GuessTheNumber</class>
 <widget class="QWidget" name="Form_GuessTheNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Apply">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_New_Game">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_enter_number">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите число:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_attempts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Попытки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_advice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Начни игру :)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_current_attempts">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
